### PR TITLE
108 upgrade react to 18

### PR DESCRIPTION
**Pull request by mnicole**
this looks like a lot but don't worry!

### Changes
-   Upgrade Storybook version to 7 * This required an update to all the stories. They have been moved from `_stories` folder to `stories` folder, so ignore all the changes in `_stories` as they are just deletes.
-   Upgrade React to 18
-   Added color swatches page
-   Deprecate CircularThumbnail, NumberCircle, ProgressBar, Trend, Avatar, Breadcrumbs, BreadcrumbsWrapper *Not used by any of our projects.
-   Update examples in documentation to use hooks *I've tested these all in codesandbox but you can check them out too
-   Update gumdrops docs
-   Update tests to React Test Library/react-test-renderer since Enzyme is deprecated

### To Test
- Go to the build link and ensure each of the components works as expected https://feature-108-upgrade-react--gumdrops.netlify.app/

### CODE REVIEW

File: stories/Colors.mdx
Comment (morganee) [inline stories/Colors.mdx:1]: Love this !

File: stories/atoms/LoadingDots.stories.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { Title, Description, Primary, Source, ArgsTable } from '@storybook/addon-docs';
+import LoadingDots from '../../components/atoms/LoadingDots';
+
+const sizeOptions = ['', 'sm', 'lg'];
```
Comment (morganee) [inline stories/atoms/LoadingDots.stories.js:4]: Is it expected that the first option is empty ?

File: stories/molecules/Checkbox.mdx [hunk 2]
```diff
@@ -54,8 +55,8 @@ render() {
                 { /* Pass defaultChecked if you want
                 the input to be checked initially */ }
                 <Checkbox
-                    name="parrots"
-                    label="party parrots"
+                    name="pineapple"
+                    label="pineapple on pizza?"
```
Comment (morganee) [inline stories/molecules/Checkbox.mdx:59]: Yes

File: stories/utilities/AbsolutePosition.mdx
[the remark this answers is not in the document]
Comment (mnicole) [reply stories/utilities/AbsolutePosition.mdx:1]: ooh good find, thanks! i forgot to check all these after i made the changes. fixed now!

File: stories/atoms/LoadingDots.stories.js (hunk 1 above)
Comment (mnicole) [reply stories/atoms/LoadingDots.stories.js:4]: yes, it makes it so you can test applying no option (in this case, if you don't pass a size props, it'll render as what i'd call medium)

**Review by morganee — APPROVED** [review]

**Pull request merged**